routes/web.php

```diff
@@ -26,6 +26,9 @@
         Route::get('/', function () {
             return Inertia::render('Item/Index', ['data' => ['yuto', 'hayashi', 'shu', 'sakadume', 'oowada'], 'user' => Auth::user()]);
         })->name('index');
+        Route::get('/detail/{item_id}', function ($item_id) {
+            return '/detail/' . $item_id;
+        })->name('detail');
```

Comment:
```suggestion
        Route::resource('/', ItemController::class)->names([
            'index' => '',
            'create' => 'create',
            'store' => 'store',
            'show' => 'show',
            'update' => 'update',
            'destroy' => 'destroy',
        ]);
```

@Apro-yuto 書き方色々あると思いますが、リソースコントローラー使うイメージだったので、こんな感じになるのかと。  
group()の内側の記述がやや不安で間違ってるかもしれませんが...
どうでしょうか？

Reply:
@YutaManaka 
結論ここの書き方はどう書いてもいいかなと思っていて、
今回はあえて可読性をあげる目的でgroupでまとめてそれぞれをrouteファイルに記述しています。

ただ、自分がメソッドを調べたりするときにいちいちphp artisan route:listしちゃうからこう実装しただけなので、
resourceコントローラを使用しても大丈夫です。

Reply:
@Apro-yuto 遅くなりました、了解です！
確かにgroup()使うとweb.phpの可読性は上がって見やすいですねー。
勉強になります！
では元の記述でOKだと思いますので、マージしちゃってOKだと思います。

Reply:
@YutaManaka 
お忙しい中、確認ありがとうございます。

resourceコントローラ使用するとプログラムの行数がコンパクトになるのでそちらもすごく魅力的なんですが、
自分は記憶力が死ぬほど悪いので、Group使ってますw

ありがとうございます！マージします！！